MinimalClient/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "msal-react-quickstart",
+  "version": "0.1.0",
+  "description": "A React single-page application calling Microsoft Graph API using MSAL.js (w/ AAD v2 endpoint) integrated with the ACS Auth sample",
```

Comment:
Maybe worth mentioning that this client is  used to give users E2E experience for our ACS Authentication Server sample